libretroshare/src/gxs/gxssecurity.cc

```diff
@@ -1112,26 +1112,20 @@ bool GxsSecurity::validateNxsGrp(const RsNxsGrp& grp, const RsTlvKeySignature& s
 		// copy msg data and meta in allmsgData buffer
 		if(grp.grp.bin_data && grp.grp.bin_len>0)
 		{
```

Comment:
No! The metadata need *always* to be copied. As I told you: only the grp.grp.bin_data.

Reply:
Done.